workshops/management/commands/report_all_instructor_activity.py

```diff
@@ -1,57 +0,0 @@
-import sys
```

Comment:
`mail-all-activity.py` is being merged with `report_all_instructor_activity.py` command into `instructors_activity` command.